src/booleans.js

```diff
@@ -1,62 +1,122 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  //   if (a === true && b === true) {
+  //     return true;
+  //   }
+  //   return false;
+  // }
+  return a === true && b === true;
```

Comment:
You dont need to explicitly check truthiness like this, remember all values in JS are either truthy or falsy so you can just compare them directly in cases like this:
```suggestion
  return a && b;
```

Also the comments should be removed